cmd/osctl/cmd/install_linux.go

```diff
@@ -69,9 +69,9 @@ var installCmd = &cobra.Command{
 		}
 
 		sequence := runtime.None
-		if upgradeArg {
-			sequence = runtime.Upgrade
-		}
+		// if upgradeArg {
+		// 	sequence = runtime.Upgrade
+		// }
```

Comment:
There is a bug that I haven't found yet that is making the initial install and upgrade sequence. We have an if statement in the etcd service that assumes if an upgrade has been performed, then it attempts to discover current control plane node IPs using the Kubernetes API. The problem is that it thinks an upgrade has been performed, so when attempting to discover the other members it fails.

Reply:
I plan on fixing in a follow up to keep this scoped as a test.